gsc/explore_lc_orbit.py

```diff
@@ -145,10 +145,10 @@ def queued_orbit_search(init_graph, local_ops, save_edges, verbose):
         visited += 1
         # Gets next graph on queue and finds representative nodes
         graph_label = queue.pop()
-        graph = class_graph.node[graph_label]['nx_graph']
```

Comment:
It seems the deprecation of `node` in favour of `nodes` is a NetworkX 2.0 feature, so we should peg the version in `setup.py` to `networkx>=2.0.0`.